common/src/main/java/org/dash/wallet/common/ui/payment_method_picker/PaymentMethodPicker.kt

```diff
@@ -90,6 +95,7 @@ class PaymentMethodPicker(context: Context, attrs: AttributeSet): ConstraintLayo
                     showSearch = false
                 ) { _, index, dialog ->
                     dialog.dismiss()
+                    analytics.logEvent(AnalyticsConstants.Coinbase.CHANGE_PAYMENT_METHOD, bundleOf())
```

Comment:
This is the same mistake as before with the `EnterAmount` fragment. This component should not care about analytics, and it might be used in other integration modules. `AnalyticsService` should be injected.